File: JavaScript/8js 1/converter.js
/*
|--------------------------------------------------------------------------
| Miles per gallon to kilometers per liter
|--------------------------------------------------------------------------
|
*/

function converter(mpg) {
   return Number(((mpg * 1.609344) / 4.54609188).toFixed(2));
}


function converter(mpg) {
   return +(mpg * 0.354006043538).toFixed(2);
}


function converter(mpg) {
   return Math.round(mpg * 35.400604353821375031) / 100;
}


console.log(converter(10), 3.54);
console.log(converter(20), 7.08);
console.log(converter(30), 10.62);

File: Python/8py 1/converter.py
"""
|--------------------------------------------------------------------------
| Miles per gallon to kilometers per liter
|--------------------------------------------------------------------------
|
"""

def converter(mpg):
   return round(mpg * (1.609344/4.54609188), 2)


converter = lambda x: round(x * 0.354006043538, 2)


def converter(mpg):
   LITTER_PER_GALLON = 4.54609188
   KILOMETRES_PER_MILE = 1.609344
   return round(mpg * (KILOMETRES_PER_MILE/LITTER_PER_GALLON), 2)


print(converter(12), 4.25)
print(converter(24), 8.50)
print(converter(36), 12.74)